README.md

```diff
@@ -97,6 +97,13 @@ More advanced features like complex application analysis or some edge cases. Thi
 $ make test-tier2
 ```
 
+### Tier 3
+Private cases of Tier 1. This tier executes tests which include sensetive data (test configuration)
```

Comment:
We can change this to : 

This tier tests require credentials for some apps and will only be run through jenkins .These should be excluded from PR runs .